doc/nrf/ug_bootloader_external_flash.rst

```diff
@@ -3,13 +3,57 @@
 Using external flash memory partitions
 ######################################
 
+
 When using MCUboot, you can store the storage partition for the secondary slot in the external flash memory, using a driver for the external flash memory that supports the following features:
 
 * Single-byte read and write.
 * Writing data from the internal flash memory to the external flash memory.
 
+.. note::
+
+   The Partition Manager will only support run-time access to flash partitions defined in regions placed on placed on external flash devices that have drivers compiled in.
+   Partition Manager has no way to determine whether partitions are used at run-time or not during compilation, but it will decide to not include partition into map if there is no driver enabled for it.
+   Lack of partition access will cause MCUboot to fail at run-time.
+   For more details on configuring and enabling access to external flash devices please refer to :ref:`pm_external_flash`.
```

Comment:
```suggestion
.. note::

   The Partition Manager only supports runtime access to flash memory partitions defined in regions placed on external flash devices that have drivers compiled in.
   Partition Manager can not determine whether partitions are used during runtime or compilation, but it does not include partitions in the map when there are no drivers enabled for them.
   Lack of partition access causes MCUboot to fail during runtime.
   
   For more details on configuring and enabling access to external flash devices, see :ref:`pm_external_flash`.
```

Reply:
Still will need to fix this line

```
Partition Manager can not determine whether partitions are used during runtime or compilation, but it does not include partitions in the map when there are no drivers enabled for them.
```

as what it actually means is that partition manager configuration is used for compilation to provide partitions but it does not mean they are used, and partition manager does not know whether they are used or not. This means that we are not able to report, at compilation, that some of partitions used in code have been left out of partition map as there are no device drivers for flash they are on.